yoyodyne/models/modules/transformer.py

```diff
@@ -336,21 +336,21 @@ def forward(
                 encoder.
             features_memory (torch.Tensor): the mask for the features.
             target_mask (Optional[torch.Tensor], optional): the mask for the
-                target sequence. Defaults to None.
+                target sequence.
```

Comment:
Should include defaults in docstrings.

Reply:
That's already documented on the previous line, that's what "optional" means in this context.